kivy/uix/gridlayout.py

```diff
@@ -308,7 +309,15 @@ def on_children(self, instance, value):
 
     @property
     def _fills_row_first(self):
-        return self.orientation[0] in 'lr'
+        return self.orientation[0] not in 'tb'
```

Comment:
Bubble over-writes `orientation`, makes it only accepts `vertical` and `horizontal`, which is why I changed this property. By this change, Bubble is always `lr-tb`.

Reply:
I think this is too sensitive to the issue of action-at-a-distance, where the exact way we do it here effect bubble. Instead, it's better if you override this in bubble by re-implementing the methods there and always return `True/False`